recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "sagemaker-code-editor" %}
-{% set version = "1.3.2" %}
+{% set version = "1.1.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
-  url: https://github.com/aws/sagemaker-code-editor/releases/download/v1.3.2/code-editor1.3.2.tar.gz
-  sha256: d538d91b4c5f9578a8ee7b40e00e3353d48281b80d239a8bab4612ff3cf9b39a
+  url: https://github.com/aws/sagemaker-code-editor/releases/download/v1.1.1/code-editorv1.1.1.tar.gz
+  sha256: f71f6c9c72b6646a5eef86593224c8fe17b1d871e0d1d64ebdbf0d574c5086dd
```

Comment:
are we not using different branches for each minor version? Managing multiple versions using one branch will be error prone

Reply:
Akash is planning to revert this change once the build has deployed to condaforge. Definitely a good idea to start a 1.1.1 branch from this commit though, once it has been merged

Reply:
I agree, this is not preferable. One question was whether this release process works with a non-main branch. We can try it with this change (I can open a new PR).

Reply:
from reading [this section](https://conda-forge.org/docs/maintainer/updating_pkgs/#forking-and-pull-requests), it does look like any branch would work

yes, let's try it out